### PR TITLE
feat(webhook): Add Webhook Examples

### PR DESCRIPTION
**Pull request by davidkathoh**
This pull request adds examples of webhook functions to the project. The new examples demonstrate how to implement and use webhooks effectively, providing clear and practical usage scenarios.

These additions aim to help developers understand and implement webhook functionality more easily.

### CODE REVIEW

File: examples/append_address_to_webhook.rs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+use helius::types::Cluster;
+use helius::Helius;
+
+#[tokio::main]
+async fn main() {
```
Comment (0xIchigo) [inline examples/append_address_to_webhook.rs:5]: This should have a return type. You could import `Result` from `use helius::error::Result` and use the type alias for the return type so the line reads `async fn main() -> Result<()> {`. You'll also have to add in `Ok(())` at the end of the block to satisfy the return type. If you get confused, I recommend looking at the other examples we currently have for the SDK

File: examples/append_address_to_webhook.rs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    let api_key: &str = "your_api_key";
+    let cluster: Cluster = Cluster::MainnetBeta;
+
+    let helius: Helius = Helius::new(api_key, cluster).unwrap();
+
+    let webhook_id = "your_webhook_id";
+    let new_addresses = ["your_address1".to_string(), "your_address2".to_string()];
+    let response = helius.append_addresses_to_webhook(webhook_id, &new_addresses).await;
```
Comment (0xIchigo) [inline examples/append_address_to_webhook.rs:13]: Can you make the `response` type explicit here, please?

File: examples/create_webhook.rs [hunk 3]
```diff
@@ -0,0 +1,26 @@
+use helius::types::{Cluster, WebhookType};
+use helius::{
+    types::{CreateWebhookRequest, TransactionType},
+    Helius,
+};
+
+#[tokio::main]
+async fn main() {
```
Comment (0xIchigo) [inline examples/create_webhook.rs:8]: Same comment re `main` that was made in `examples/append_address_to_webhook.rs`. All of the example files should follow this convention

File: examples/create_webhook.rs [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        webhook_url: "your_webhook".to_string(),
+        transaction_types: vec![TransactionType::Any],
+        account_addresses: vec!["your_addresses".to_string()],
+        webhook_type: WebhookType::Enhanced,
+        auth_header: None,
+        txn_status: Default::default(),
+        encoding: Default::default(),
+    };
+
+    let response = helius.create_webhook(request).await;
```
Comment (0xIchigo) [inline examples/create_webhook.rs:24]: Can you also make this `response` type explicit, please?

Comment (0xIchigo) [reply examples/create_webhook.rs:24]: And can you please make all the remaining `response` vars in the subsequent examples explicit, please?

**Review by 0xIchigo — CHANGES_REQUESTED** [review]
Very based PR! Just a few comments regarding the `main` functions having return types and making other types more explicit. Solid content-wise though :fire:

File: examples/create_webhook.rs (hunk 4 above)
Comment (davidkathoh) [reply examples/create_webhook.rs]: It should be good now

File: examples/append_address_to_webhook.rs [hunk 5]
```diff
@@ -1,15 +1,18 @@
-use helius::types::Cluster;
+use helius::error::HeliusError;
+use helius::types::{Cluster, Webhook};
 use helius::Helius;
 
 #[tokio::main]
-async fn main() {
+async fn main() -> Result<(), HeliusError> {
```
Comment (0xIchigo) [inline examples/append_address_to_webhook.rs:6]: Very nice! Nit: we have a [`Result` type alias](https://github.com/helius-labs/helius-rust-sdk/blob/6a9a7962250810466bef296ee511578fa3c1940e/src/error.rs#L141-L142) that can be used here to simplify the code from `Result<(), HeliusError>` to `Result<()>`. Can you please update the examples to this?

**Review by 0xIchigo — CHANGES_REQUESTED** [review]

File: examples/append_address_to_webhook.rs (hunk 5 above)
Comment (davidkathoh) [reply examples/append_address_to_webhook.rs]: Gotcha!!

**Review by 0xIchigo — APPROVED** [review]
LGTM 🔥

**Pull request merged**